views/SVGs/Eventos/Eventos.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
```

Comment:
Please convert this to an SVG, like views\Shared\Layouts\Dashboard\index.tsx line 12

Reply:
Usually is better the react way of having a component like this:

- Size reduction: SVG-to-React component conversion shaves off some excess SVG styling.
- No extra requests are made by the browser to fetch SVG images since they are now bundled with the JS. In the case that you point out, the SVG is bundled so you don't win speed anyway.
- It allows for easier control and maintenance of SVGs since they are now React components.
- Verbose and obscure SVGs can be converted to customizable React components that expose a simple declarative API.

I think the current way is better